mne_bids/tests/test_read.py

```diff
@@ -125,3 +128,35 @@ def test_handle_events_reading():
 
     raw = _handle_events_reading(events_fname, raw)
     events, event_id = mne.events_from_annotations(raw)
+
+
+def test_empty_room():
+    """Test reading of empty room data."""
+    bids_root = _TempDir()
+
+    raw = mne.io.read_raw_fif(raw_fname)
+    bids_basename = make_bids_basename(subject='01', session='01',
+                                       task='audiovisual', run='01')
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+
+    er_fname = get_matched_empty_room(bids_basename + '_meg.fif',
+                                      bids_root)
+    assert er_fname is None
+
+    er_raw_fname = op.join(data_path, 'MEG', 'sample', 'ernoise_raw.fif')
+    er_raw = mne.io.read_raw_fif(er_raw_fname)
+    er_date = datetime.fromtimestamp(
+        er_raw.info['meas_date'][0]).strftime('%Y%m%d')
```

Comment:
this is tricky to get write especially if you anonymize. Public API should simplify this.

Reply:
Let's make an example. I think it will clarify what are the pain points.